kratos/sources/kratos_parameters.cpp

```diff
@@ -228,52 +231,154 @@ Parameters::Parameters(const std::string& rJsonString)
 {
     mpRoot = Kratos::make_shared<nlohmann::json>(nlohmann::json::parse( rJsonString, nullptr, true, true));
     mpValue = mpRoot.get();
-    SolveIncludes(*mpValue); 
+
+    // Initialize the include graph
+    std::map<std::string,std::set<std::string>> adjacency_list;
```

Comment:
Do we really need this map? The stack itself is not enough?

Reply:
the ```stack``` is used for parsing the ```JSON```, the ```map``` is needed for keeping track of which file includes which other files, to detect infinite circular includes.